src/Form/Form.js

```diff
@@ -19,7 +19,7 @@ import {
     formatCCYear,
     getCountryName,
     prepareFormValuesForStorage,
-    inferCountryCodeFromElement
+    inferCountryCodeFromElement, inferElementLocale
```

Comment:
NAB: Align formatting with rest of the list
```suggestion
    inferCountryCodeFromElement,
    inferElementLocale
```